src/index.html

```diff
@@ -20,8 +21,393 @@
       href="styles/index.scss"
     />
   </head>
+  <!-- #endregion -->
 
   <body>
-    <h1>Catalog</h1>
+    <!-- #region header  -->
+    <header class="header">
+      <div class="header__logo-container">
+        <a
+          href="index.html"
+          class="header__logo-link"
+        >
+          <img
+            src="images/logo.png"
+            alt="MOYO logo"
+          />
+        </a>
+      </div>
+
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a
+              href="#apple"
+              class="header__nav-link is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="header__nav-item">
```

Comment:
Add the blank line between the links
```suggestion
          </li>
          
          <li class="header__nav-item">
```